dev-docs/publisher-api-reference.md

```diff
@@ -1290,4 +1285,79 @@ This call returns the following DFP video ad tag URL:
+
+Reasons you might pass the `options.url` field include:
+
+- You are not using an ad server, and you want to insert the winning VAST content into your player directly
+- You are using a video ad server that can accept a standalone VAST URL, as opposed to having to pass the VAST URL as a query string parameter  
+
+The table below lists the expected behavior of this method when called with various combinations of arguments and cache usage.
+
+{: .table .table-bordered .table-striped }
+| Argument                           | Prebid Cache Enabled? | Notes                                                                                                                                                                                    |
+|------------------------------------+-----------------------+------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|
+| `options.params` only              | Yes                   | None.                                                                                                                                                                                     |
+| `options.params` only              | No                    | Bidder's response must contain `bid.vastUrl` for video ad to serve.                                                                                                                      |
+| `options.url` only                 | Yes                   | Prebid attaches the bid's adserver targeting and rebuilds the URL based on the input by merging publisher input with default values. It does not set `description_url` to `bid.vastUrl`. |
+| `options.url` only                 | No                    | Bidder's response must contain `bid.vastUrl` for video ad to serve.  Prebid sets `description_url` to `bid.vastUrl`.                                                                     |
+| `options.params` and `options.url` | Yes                   | Prebid attaches the bid's adserver targeting and rebuilds the URL based on the input by merging publisher input with default values. It does not set `description_url` to `bid.vastUrl`. |
+| `options.params` and `options.url` | No                    | Bidder's response must contain `bid.vastUrl` for video ad to serve.                                                                                                                      |
+
+{: .alert.alert-info :}
+**Bid Responses and Prebid Cache**  
+Video bidders must supply either `bid.vastUrl` or `bid.vastXml`.  They may supply both.  If the video bidder supplies *only* `bid.vastXml` and Prebid Cache is disabled, this will not work and the bid will be dropped from the auction (if Prebid Cache is enabled, this same scenario is fine). If a video bidder supplies neither `bid.vastUrl` or `bid.vastXml`, the bid is dropped.
```

Comment:
I think we can get rid of the last sentence: "If a video bidder supplies neither `bid.vastUrl` or `bid.vastXml`, the bid is dropped.", as it seems repetitive.